README.md

```diff
@@ -87,97 +93,141 @@ However, this is always optional, and usually not necessary.
 " Creates a new note and uses the last visual selection as the title while replacing the selection with a link to the new note
 " params
 "   (optional) additional options, see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zknew
-:ZkNewLink [<options>]
+:'<,'>ZkNewFromTitleSelection [<options>]
+
+" Creates a new note and uses the last visual selection as the content while replacing the selection with a link to the new note
+" params
+"   (optional) additional options, see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zknew
+:'<,'>ZkNewFromContentSelection [<options>]
+
+" cd into the notebook root
+:ZkCd
+
+" Opens a Telescope picker
+" params
+"   (optional) additional options, see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+:ZkNotes [<options>]
+
+" Opens a Telescope picker
+" params
+"   (optional) additional options, see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+:ZkBacklinks [<options>]
 
 " Opens a Telescope picker
 " params
 "   (optional) additional options, see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
-:ZkList [<options>]
+:ZkLinks [<options>]
+
+" Opens a Telescope picker, filters for notes that match the text in the last visual selection
+" params
+"   (optional) additional options, see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+:'<,'>ZkMatch [<options>]
 
 " Opens a Telescope picker
 " params
 "   (optional) additional options, see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zktaglist
-:ZkTagList [<options>]
+:ZkTags [<options>]
 ```
 where `options` can be any valid *Lua* expression that evaluates to a table.
 
 *Examples:*
 ```vim
 :ZkNew { dir = "daily", date = "yesterday" }
 :ZkList { createdAfter = "3 days ago", tags = { "work" } }
-:'<,'>ZkNewLink " this will use your last visual mode selection. Note that you *must* call this command with the '<,'> range.
+:'<,'>ZkNewFromTitleSelection " this will use your last visual mode selection. Note that you *must* call this command with the '<,'> range.
+:ZkCd
 ```
 
 ### Lua
 ```lua
 ---Indexes the notebook
 --
----@param path? string path to explicitly specify the notebook
 ---@param options table additional options
+---@param path? string path to explicitly specify the notebook
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zkindex
-require("zk").index(path, options)
+require("zk.commands").index(options, path)
```

Comment:
Why did you move the main functions under a `zk.commands` module? I think keeping it at the root level would make it more convenient to setup key bindings and make it clearer what are the user high-level functions.